src/Goerli-DssSpellCollateral.sol

```diff
@@ -32,79 +35,61 @@ contract DssSpellCollateralAction {
-        //         join:                  MCD_JOIN_XXX_A,
-        //         clip:                  MCD_CLIP_XXX_A,
-        //         calc:                  MCD_CLIP_CALC_XXX_A,
-        //         pip:                   PIP_XXX,
-        //         isLiquidatable:        BOOL,
-        //         isOSM:                 BOOL,
-        //         whitelistOSM:          BOOL,
-        //         ilkDebtCeiling:        line,
-        //         minVaultAmount:        dust,
-        //         maxLiquidationAmount:  hole,
-        //         liquidationPenalty:    chop,
-        //         ilkStabilityFee:       duty,
-        //         startingPriceFactor:   buf,
-        //         breakerTolerance:      tolerance,
-        //         auctionDuration:       tail,
-        //         permittedDrop:         cusp,
-        //         liquidationRatio:      mat,
-        //         kprFlatReward:         tip,
-        //         kprPctReward:          chip
-        //     })
-        // );
-
-        // DssExecLib.setStairstepExponentialDecrease(
-        //     CALC_ADDR,
-        //     DURATION,
-        //     PCT_BPS
-        // );
-
-        // DssExecLib.setIlkAutoLineParameters(
-        //     "XXX-A",
-        //     AMOUNT,
-        //     GAP,
-        //     TTL
-        // );
-
+        //  Add RETH-A as a new Vault Type
+        //  Poll Link: [https://vote.makerdao.com/polling/QmfMswF2]
+
+        DssExecLib.addNewCollateral(
+            CollateralOpts({
+                ilk:                   "RETH-A",
+                gem:                   RETH,
+                join:                  MCD_JOIN_RETH_A,
+                clip:                  MCD_CLIP_RETH_A,
+                calc:                  MCD_CLIP_CALC_RETH_A,
+                pip:                   PIP_RETH,
+                isLiquidatable:        true,
+                isOSM:                 true,
+                whitelistOSM:          true,
+                ilkDebtCeiling:        5 * MILLION,
+                minVaultAmount:        15 * THOUSAND,                // debt floor - dust in DAI
+                maxLiquidationAmount:  2 * MILLION,
+                liquidationPenalty:    1300,                         // 13% penalty on liquidation
+                ilkStabilityFee:       ONE_FIVE_PCT_RATE,            // 1.50% stability fee
+                startingPriceFactor:   11000,                        // Auction price begins at 110% of oracle price
+                breakerTolerance:      5000,                         // Allows for a 50% hourly price drop before disabling liquidation
+                auctionDuration:       120 minutes,
+                permittedDrop:         4500,                         // 45% price drop before reset
+                liquidationRatio:      17000,                        // 170% collateralization
+                kprFlatReward:         250,                          // 300 DAI tip - flat fee per kpr
```

Comment:
Maybe just remove the value to prevent this issue in the future?

```suggestion
                kprFlatReward:         250,                          // DAI tip - flat fee per kpr
```